Document connector templates in desktop modeler (#4053)
camunda/camunda-docs: docs/components/modeler/desktop-modeler/element-templates/using-camunda-connector-templates.md

> ---

Reviewer: I propose we move this section out of the element templates section. It is strictly "user documentation" and not related to element templates.

> id: using-camunda-connector-templates

Reviewer: ```suggestion
id: using-connectors
```

title: Using Camunda Connector templates

> description: "Learn how to use Camunda Connector templates in Desktop Modeler."

Reviewer: ```suggestion
description: "Learn how to use Camunda Connectors from Desktop Modeler."
```

Reviewer: Consider: Add a badge `Camunda 8 only`.

---

> Users of Camunda 8 and Desktop Modeler can use the same [Connector templates](../../../connectors/out-of-the-box-connectors/available-connectors-overview.md) that are available in Web Modeler.

Reviewer: ```suggestion
You can enable automatic fetching of [Camunda Connector templates](../../../connectors/out-of-the-box-connectors/available-connectors-overview.md) in the Desktop Modeler. This will make all out-of-the-box templates available for direct use.

Alternatively connector templates may be configured globally or per project following the standard configuration mechanisms for element templates.  
```

> ## Enable Camunda 8 Connector templates

Reviewer: ```suggestion
## Enable automatic fetching of Connector templates
```

Camunda 8 Connector templates can be enabled through the `enable-connector-templates` [flag](../flags/flags.md). When enabled the templates will be kept up to date and you'll be able to use them in the Camunda 8 BPMN editor.

On startup a notification will indicate whether the templates are up to date or have been updated.

![Camunda Connector templates up to date notification](./img/using-camunda-connector-templates/up-to-date.png)

In case of an error you'll see a nofication, too.

![Error updating Camunda Connector templates notification](./img/using-camunda-connector-templates/error.png)

With the templates enabled, you'll be able to use them in the Camunda 8 BPMN editor.

![Using Camunda Connector templates in the Camunda 8 BPMN editor](./img/using-camunda-connector-templates/apply.png)

> ## Additional resources

Reviewer: I propose that we add an additional section to allow people to learn that they can "configure / manually download" connector templates of their choice using standard discovery mechanism (link to `configure-element-templates`).
